cmd/core-data/main.go

```diff
@@ -45,6 +47,20 @@ func main() {
 	flag.Parse()
 
 	httpServer := handlers.NewServerBootstrap(data.LoadRestRoutes())
+
+	bootstrapHandlers := []interfaces.BootstrapHandler{
+		data.NewServiceInit(&httpServer).BootstrapHandler,
+		telemetry.BootstrapHandler,
+		httpServer.Handler,
+		handlers.NewStartMessage(clients.CoreDataServiceKey, edgex.Version).Handler,
+	}
+
+	// Unless the secret store functionality has been explicitly disabled, add the SecretClient bootstrap handler as the
+	// first handler since it will be needed by other bootstrap handlers.
+	if env := os.Getenv(internal.SecretStore); env != "false" {
+		bootstrapHandlers = append([]interfaces.BootstrapHandler{handlers.SecretClientBootstrapHandler}, bootstrapHandlers...)
+	}
```

Comment:
This is an implementation detail that will be common across all services.  Can you revert your changes here, always run the handlers.SecretClientBootstrapHandler, and let it determine in a common location whether or not to do anything based upon the presence or absence of the environment variable?